assets/set_variables.sh

```diff
@@ -96,6 +111,16 @@ function is_commit_based {
   return 1
 }
 
+# this is to detect file system builds
+function is_file_system_based {
+  local check_version=$1
+  if [ -d $check_version ]
+  then
+    return 0
+  fi
+  return 1
```

Comment:
The return value from the `-d` test is the same as the return statements in the function, so the conditional can be removed:

```suggestion
  [[ -d $check_version ]]
```